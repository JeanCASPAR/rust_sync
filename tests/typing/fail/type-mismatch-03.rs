use rust_sync::sync;

sync! {
    node oui() = (a)
    where
        a : int = false -> !(pre a);
}

fn main() {}
