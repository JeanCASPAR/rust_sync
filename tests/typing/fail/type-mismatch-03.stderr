error: type mismatch: expected `bool`, found `int`
 --> tests/typing/fail/type-mismatch-03.rs:6:28
  |
6 |         a : int = false -> !(pre a);
  |                            ^^^^^^^^
